sql/activity_stream/impression_stats_by_experiment/view.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.activity_stream.impression_stats_by_experiment`
```

Comment:
@kirill-demtchouk - This view contains the logical tile type info and would be the target for all your experiment analysis queries. Does this name look good?

Reply:
Looks good! I see Mozilla uses Airflow too 👍 

One question on permissions for this new table: 

In addition to Redash access, can we also make sure that this table is accessible from our BI tool (Mode Analytics)? We use this service account for that tool: external-mode@moz-fx-data-bq-pocket.iam.gserviceaccount.com.

Reply:
That service account is granted read access to the `activity_stream_bi` dataset, so any new tables defined there will be accessible. This view won't be accessible, but the underlying `impression_stats_by_experiment_v1` table will be. Does that meet your needs? If having the view would be useful, we can duplicate it in `activity_stream_bi`.

Reply:
That should be fine. The only difference between the view and the table is the join against `tile_id_types`.